library/network/src/modules/SuSEFirewall.rb

```diff
@@ -678,6 +678,15 @@ def IsVerbose
       Ops.greater_than(@verbose_level, 0)
     end
 
+    # Local function returns if protocol is supported by firewall.
+    # Protocol name must be in upper-cases.
+    #
+    # @param [String] protocol
+    # @return	[Boolean] if protocol is supported
```

Comment:
Oh, I have not noticed at first: SF2 allows RPC but FWD does not. That should have been mentioned explicitly.
Again, the code using this library simply [expects RPC to be supported](https://github.com/yast/yast-firewall/search?utf8=%E2%9C%93&q=RPC).